setup.py

```diff
@@ -61,7 +61,7 @@
     install_requires=[
         "importlib-metadata ; python_version<'3.8'",
         "py-solc-x>=1.1.0,<1.2.0",
-        "eth-ape>=0.1.0a15",
+        "eth-ape>=0.1.0a23",
```

Comment:
Ha just now seeing this, might as well switch to `0a25`

Reply:
I did switch to 25 but now i'm having issues passing pytest check